ProcessList.c

```diff
@@ -25,6 +26,8 @@ ProcessList* ProcessList_init(ProcessList* this, const ObjectClass* klass, Users
    // tree-view auxiliary buffer
    this->processes2 = Vector_new(klass, true, DEFAULT_SIZE);
 
+   this->treeSet = Hashtable_new(140, false);
```

Comment:
Any chance you could better guess-timate this initial size?

As an hashtable should usually be filled to 2/3 or 3/4 and we know the rought number of items we have, we might work from there?

Reply:
at `ProcessList_init` time it's impossible?
But at the time we use the map we know the exact size (`size == Vector_size(this->processes)`).

Reply:
k, mea culpa ;-) Thought we had some Vector there already to derive this information.
Anyway, 140 is quite optimistic: Just on my current system I reacht about 1000 threads and I have hardly anything™ open (except for chromium+some shells). Going from there and knowing that we basically have to manage a few pointers we could do 4096 for the initial size of that HashSet and will most likely stay within the constraints for the HashSet to keep good performance.